spark-connector/src/main/java/com/datastrato/gravitino/spark/GravitinoSparkConfig.java

```diff
@@ -0,0 +1,15 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark;
+
+public class GravitinoSparkConfig {
```

Comment:
You should add a private constructor for this class if it only used to manage static variables.

Reply:
done